Compiling error in the last Manjaro version

RigelEngine/src/engine/random_number_generator.cpp: En la función miembro ‘int rigel::engine::RandomNumberGenerator::gen()’:
RigelEngine/src/engine/random_number_generator.cpp:53:59: error: ‘size_t’ no nombra a un tipo
   53 |   static_assert(RANDOM_NUMBER_TABLE.size() == static_cast<size_t>(std::numeric_limits<decltype(mNextNumberIndex)>::max()) + 1u,
      |                                                           ^~~~~~
RigelEngine/src/engine/random_number_generator.cpp:20:1: nota: ‘size_t’ is defined in header ‘<cstddef>’; did you forget to ‘#include <cstddef>’?
   19 | #include <limits>
  +++ |+#include <cstddef>
   20 | #include <type_traits>
make[2]: *** [src/CMakeFiles/rigel_core.dir/build.make:342: src/CMakeFiles/rigel_core.dir/engine/random_number_generator.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:501: src/CMakeFiles/rigel_core.dir/all] Error 2
make: *** [Makefile:146: all] Error 2

--- a/src/engine/random_number_generator.cpp
+++ b/src/engine/random_number_generator.cpp
@@ -16,8 +16,10 @@
 
 #include "random_number_generator.hpp"
 
+#include <cstddef>
 #include <limits>
 #include <type_traits>
+
 
 namespace rigel::engine {
 

